Sources/BraintreeApplePay/BTApplePayCardNonce.swift

```diff
@@ -10,11 +10,18 @@ import BraintreeCore
     /// The BIN data for the card number associated with this nonce.
     public let binData: BTBinData
 
+    /// This Boolean indicates whether this tokenized card is a device-specific account number (DPAN) or merchant/cloud token (MPAN). Available on iOS 16+.
+    /// If `isDeviceToken` is `false`, then token type is MPAN
+    public var isDeviceToken: Bool
+
     init?(json: BTJSON) {
         guard let nonce = json["nonce"].asString() else { return nil }
 
         let cardType = json["details"]["cardType"].asString() ?? "ApplePayCard"
         let isDefault = json["default"].isTrue
+        let isDeviceToken = json["details"]["isDeviceToken"].asBool() ?? true
+
+        self.isDeviceToken = isDeviceToken
```

Comment:
I think we just extract the properties above because it'd make the super init a mess. Here I think we can do:
```suggestion

        self.isDeviceToken = json["details"]["isDeviceToken"].asBool() ?? true
```